### ISSUE TITLE
Document when/how to review EventLog from handle_skipped_award

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/credere-backend#369]
We create these rows when SkippedAwardError is raised. Presumably, we do so in order to review/monitor those messages.

**Comment by yolile** [comment]
> Presumably, we do so in order to review/monitor those messages.

Yes, the main use case is to monitor how many invitations are not being sent because of data quality issues. Additional use cases include monitoring if the invitation process is working as expected and/or if a data source was changed. I also check this table when running commands manually (e.g. fetch awards within a period or by supplier ID)

We needed the first more during the piloting stage of the project, but I still check for this when reporting to the donor

**Comment by jpmckinney** [comment]
Thanks - can you draft a new section in deployment.rst, with some examples of the SQL statements you run?
